src/conn/conn_sweep.c

```diff
@@ -121,6 +122,9 @@ __sweep_expire(WT_SESSION_IMPL *session)
 
 	conn = S2C(session);
 
+	if (conn->sweep_idle_time == 0)
```

Comment:
Add a comment - it's not obvious why an idle time of 0 means there is no work to do.